plugins/entando-plugin-jacms/src/main/webapp/WEB-INF/plugins/jacms/apsadmin/jsp/content/bulk/category/category-choice.jsp

```diff
@@ -101,17 +101,19 @@
                     <tr>
                         <th>
                             <s:text name="label.category.tree"/>
-                            <button type="button" class="btn-no-button expand-button"
-                                    id="expandAll">
-                                <span class="fa fa-plus-square-o treeInteractionButtons"
-                                      aria-hidden="true"></span>&#32;
-                                <s:text name="label.category.expandAll"/>
-                            </button>
-                            <button type="button" class="btn-no-button" id="collapseAll">
-                                <span class="fa fa-minus-square-o treeInteractionButtons"
-                                      aria-hidden="true"></span>&#32;
-                                <s:text name="label.category.collapseAll"/>
-                            </button>
+                            <s:if test="%{#categoryTreeStyleVar == 'classic'}">
```

Comment:
@zonia3000 What happens here if the tree style is not classic? This looks like it would disable tree expansion completely for those cases?

Reply:
Yes, is what I wanted, since it happens for all the other trees.
See for example [this filehttps://github.com/entando/entando-core/blob/master/admin-console/src/main/webapp/WEB-INF/apsadmin/jsp/category/entryCategory.jsp], it has the same if.

Reply:
Ok thanks. Merging